main_f7.c

```diff
@@ -830,6 +830,11 @@ main(void)
 		try_boot = false;
 	}
 
+	/* Don't leave the pulldown running forever */
```

Comment:
USE_VBUS_PULL_DOWN - was a workaround for a specific F7 HW platform with high leakage on PA9 due to wiring and component choices (not capacitance). This is atypical! 

I am not sure we want this all time for real VBUS sense (PA9) on an F7.  Please refer to F1,F4,F7 data sheet footnotes for VBUS recommendations.  

I can see why you may want it on other GPIO (not real VBUS), to eliminate a floating input  read as a one without a high value R Pull down on the board.  So maybe this needs a conditional RELEASE_VBUS_PULL_DOWN?